flask-redis/Dockerfile

```
@@ -0,0 +1,5 @@
+FROM python:3.11.0a6-alpine3.15
+ADD . /code
+WORKDIR /code
+RUN pip install -r requirements.txt
+CMD python app.py
```

To use properly layers caching we should start by copying `requirements.txt` then `pip install` and after copy the most moving part: the source code

```suggestion
FROM python:3.11.0a6-alpine3.15
WORKDIR /code
COPY requirements.txt /code
RUN pip install -r requirements.txt --no-cache-dir
COPY . /code
CMD python app.py
```